### DISCUSSION TITLE
No Fluid Structure Interaction Module in Moose install

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#17987] [Q&A Modules: General] [answered]
I would like to use the Fluid Structure Interaction module in MOOSE. When I attempted to run an example fsi model in Peacock I received an error that my install did not have the appropriate files to run an fsi model. My install of MOOSE does not have the Fluid Structure Interaction Module installed in the Modules directory. What do I need to do in order to install this application?

Thanks for any help solving this problem.

**Comment by GiudGiud** [comment, marked as answer]
Fluid structure interaction is under the modules/fsi folder. If it's not there then you need to update moose

**Reply by TLWise** [reply]
Complete. I had to perform a "git pull" update of MOOSE. Thank you for the advice.
